config/locales/en.yml

```diff
@@ -94,7 +94,7 @@ en:
         favicon: Favicon
         favicon_info_text: "Favicon will replace the default Sharetribe logo. The dimensions should be 32x32 pixels and the uploaded image will be resized to these dimensions."
         community_custom_color1: "Main color"
-        community_custom_color2: "Second color"
+        community_custom_color2: "\"Post a new listing\" button color"
```

Comment:
Maybe changing to italics?

This file:
community_custom_color2: "%{post_a_new_listing} button color"
post_a_new_listing: "Post a new listing"

edit_look_and_feel.haml(line 57):
    .row
      .col-12
        - post_a_new_listing = "<i>#{t(".post_a_new_listing_button")}<i>"
        = form.label :custom_color2, t(".community_custom_color2", post_a_new_listing: post_a_new_listing), :class => "input"